napalm/ios/ios.py

```diff
@@ -2103,6 +2105,14 @@ def get_interfaces_counters(self):
                     match = re.search(r"(\d+) output errors", line)
                     counters[interface]["tx_errors"] = int(match.group(1))
                     counters[interface]["tx_discards"] = -1
+
+            interface_type, interface_number = split_interface(interface)
+            if interface_type in [
+                'HundredGigabitEthernet',
```

Comment:
Okay, only Black is failing now (seems like it requires double quotes on these): just run `black .` (in the environment you installed `requirements-dev.txt`) and should be gtg.

Reply:
Thank you for beeing such a help :)